sources/org.osbuild.skopeo

```diff
@@ -132,9 +132,7 @@ class SkopeoSource(sources.SourceService):
             reference = f"{specifier}{imagename}@{digest}"
             source = self.get_source(transport, reference)
 
-            # We use the dir format because it is the most powerful in terms of feature support and is the closest to a
-            # direct serialisation of the registry data.
-            destination = f"dir:{archive_dir}/{self.dir_name}"
+            destination = f"containers-storage:[overlay@{archive_dir}/{self.dir_name}]{imagename}"
```

Comment:
This is also not ideal, essentially this is creating a new store for every container... but refactoring this to use a single store would break backwards compatibility.

Reply:
The other issue with this is that a local copy of a container and the remote copy of the container will have `checksums` that are equal... so if you want to change the source of the container, you essentially have to clear the store before you can pull in the new container :/